xbmc/guilib/GUIFontTTFGL.cpp

```diff
@@ -139,18 +139,6 @@ void CGUIFontTTFGL::LastEnd()
 
     // Deal with vertices that had to use software clipping
     std::vector<SVertex> vecVertices(6 * (m_vertex.size() / 4));
-    SVertex *vertices = &vecVertices[0];
```

Comment:
This seems wrong. This block adds the vertices needed to vecVertives which is used in the gl calls below. There is some pointer addition happening which may be why it thinks the variable is unused?

Reply:
Yeah, theres some pointer arithmetic, but in the end vertices isnt used for anything in this HAS_GL codepath.
In the GLES equivalent, its used in the following https://github.com/xbmc/xbmc/blob/7823116bdc46bf7668b57585669a107421ff39d2/xbmc/guilib/GUIFontTTFGL.cpp#L199-L201

Whereas the GL path does https://github.com/xbmc/xbmc/blob/7823116bdc46bf7668b57585669a107421ff39d2/xbmc/guilib/GUIFontTTFGL.cpp#L149-L154

Reply:
The gl method uses a GL_ARRAY_BUFFER

https://github.com/xbmc/xbmc/blob/7823116bdc46bf7668b57585669a107421ff39d2/xbmc/guilib/GUIFontTTFGL.cpp#L147

Reply:
ah, ok, i see. pointer to the pointer to the reference. Ill drop this then. drum it up to a false positive